components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -21,6 +21,7 @@ There are different metrics to choose from, depending on what you want to achiev
 * [Events](#Events): These allow recording of e.g. individual occurences of user actions, say every time a view was open and from where.
 * [Counters](#Counters): Used to count how often something happens, say how often a certain button was pressed.
 * [Labeled metrics](#labeled-metrics): Used to record multiple metrics of the same type under different string labels, say every time you want to get a count of different error types in one metric.
+* [Timing Distributions](#Timing-Distributions): Used to accumulate and count time measurements into buckets.
```

Comment:
Let's drop the bucket term please? It doesn't seem important to users of the metric here.
s/count time measurements into buckets/count multiple time measurements/